examples/examples_turbopark/001_compare_turbopark_implementations.py

```diff
@@ -0,0 +1,189 @@
+"""Example: Compare TurbOPark model implementations
+This example demonstrates a new implementation of the TurbOPark model that is
+more faithful to the original description provided by Nygaard et al and uses
+the sequential_solver, and compares it to the existing implementation in
+Floris.
+"""
```

Comment:
Could we add here a note on where the data comparison_data comes from, is it from one of the referenced papers?  Or the website?

Reply:
The testcases were my own, so not based on literature. The comparison data comes from running Orsted's Matlab code.

Reply:
I've now added a text comment saying that the comparison data comes from running Ørsted's Matlab code.